config-templates/ibm/ibm-vpc-block-csi-driver/5.1/metadata.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "5.1",
+  "revision": "1",
+  "enabled": "true",
+  "status": "supported",
+  "description": "[Beta] - IBM VPC Block Storage CSI driver version 5.1",
+  "tags": [
+    {
+      "name": "provider",
+      "value": "IBM"
+    },
+    {
+      "name": "volume-type",
+      "value": "block"
+    },
+    {
+      "name": "provider-logo",
+      "value": "logo/ibm-vpc-block-csi-driver.svg"
+    },
+    {
+      "name": "offering-name",
+      "value": "ibm-vpc-block-csi-driver"
+    },
+    {
+      "name": "offering-type",
+      "value": "non-sds"
+    },
+    {
+      "name": "provider-url",
+      "value": "IBM knowledge center: https://www.ibm.com/support/knowledgecenter/SSRQ8T_1.4.0/csi_block_storage_kc_welcome.html"
```

Comment:
This does not look to be correct link. Please refere version 5.0 for correct link.